src/jobsautoreport/models.py

```diff
@@ -20,3 +20,13 @@ class JobState(Enum):
 class ReportInterval(Enum):
     WEEK = "week"
     MONTH = "month"
+
+
+class MachineType(Enum):
+    M3_SMALL_X86 = "m3.small.x86"
+    C3_MEDUIM_X86 = "c3.medium.x86"
+    A3_LARGE_X86 = "a3.large.x86"
+    C3_LARGE_ARM64 = "c3.large.arm64"
+    M3_LARGE_X86 = "m3.large.x86"
+    N3_XLARGE_X86 = "n3.xlarge.x86"
+    S3_XLARGE_X86 = "s3.xlarge.x86"
```

Comment:
machine type shouldn't be hardcoded as we change them (and equinix change them) quite often